.github/workflows/python-tests.yml

```diff
@@ -61,12 +61,7 @@ jobs:
       - name: Nox lint
         working-directory: clients/python
         run: |
-          if [[ ${{ matrix.session }} == "mypy" ]]; then
-            nox --python=${{ matrix.python }} ||\
-              echo "::error title='mypy failure'::Check the logs for more details"
-          else
-            nox --python=${{ matrix.python }}
-          fi
+          nox --python=${{ matrix.python }}
```

Comment:
This `|| echo ...` catches the non-zero exit status, so the step passes even if mypy fails. We don't need this custom error message, so this can be simplified.